### PR TITLE
Parameter tweaking: reward vesting and deal weight multiplier

### PR DESCRIPTION
**Pull request by ZenGround0**
1. Change `DealWeightMultiplier` to 10
2. `AddLockedFund` defaults to using a new reward spec which is different from pledge spec.

### CODE REVIEW

File: actors/builtin/miner/miner_actor.go [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ func (a Actor) AddLockedFund(rt Runtime, amountToLock *abi.TokenAmount) *adt.Emp
 			rt.Abortf(exitcode.ErrInsufficientFunds, "insufficient funds to lock, available: %v, requested: %v", availableBalance, *amountToLock)
 		}
 
-		if err := st.AddLockedFunds(store, rt.CurrEpoch(), *amountToLock, &PledgeVestingSpec); err != nil {
+		if err := st.AddLockedFunds(store, rt.CurrEpoch(), *amountToLock, &RewardVestingSpec); err != nil {
```
Comment (ZenGround0) [inline actors/builtin/miner/miner_actor.go:1353]: It seems reasonable to impose reward vesting on funds miners intentionally lock up, but perhaps there is something more sophisticated we should be doing like enforcing different specs for different callers.

Comment (anorth) [reply actors/builtin/miner/miner_actor.go:1353]: I think the prior is how cryptoecon want it. The reward is considered pledged to the network, it's just not the "initial pledge" for a sector. I think there's only one PledgeVestingSpec.

Comment (anorth) [reply actors/builtin/miner/miner_actor.go:1353]: It seems I was wrong. They actually would prefer something like #537, but for today, different specs are good.

**Comment by anorth** [comment]
After discussion with @zixuanzh I have pushed a commit setting the initial pledge vesting spec to 180+180 days.

**Pull request merged**